superwoman7/index.html

```diff
@@ -1,57 +1,144 @@
+                                    <p class='text-muted'>@{{tweet.user.screen_name}}</p>
+                                </div>
+                                <p style='margin-top: -10px;'>
+                                    <small class='text-muted'>
+                                    <i class='glyphicon glyphicon-time'></i>
+                                    &nbsp;{{tweet.created_at.substring(0,10)}}
+                                    </small>
+                                </p>
+                            </div>
+                            <div class='timeline-body'
+                                 ng-bind-html='tweet.text'
+                                 style='margin-top:-8px;word-wrap: break-word;'>
+                            </div>
+                            <div class='tweet-images'>
+                                <img ng-src='images' ng-repeat='images in tweet.images'>
+                            </div>
+                            <hr>
+                            <div class='tweet-stats' style='margin-top: -10px;'>
+                                <p class='text-muted'>{{tweet.retweet_count}} retweets</p>
+                                <p class='text-muted'>{{tweet.favourites_count}} favorites</p>
+                                <a href='{{tweet.link}}' target="_blank" class='text-muted'> Link to Tweet </a>
+                            </div>
+                        </div>
+                    </div>
+                </li>
+            </ul>
+        </div>
+    </body>
+=======
+<head>
+<meta charset="utf-8">
+<link rel="icon" type="image/png" href="../../artwork/favicon.png">
+<title>Lokalak Tweet Timeline</title>
+<script src="//ajax.googleapis.com/ajax/libs/angularjs/1.5.0-rc.1/angular.min.js"></script>
+<script src="//code.jquery.com/jquery-1.12.0.min.js"></script>
+<script src="./js/app.js"></script>
+<script src="//cdnjs.cloudflare.com/ajax/libs/angular-loading-bar/0.7.1/loading-bar.min.js"></script>
+<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+<link rel="stylesheet" href="./css/timeline.css">
+<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/angular-loading-bar/0.7.1/loading-bar.min.css" media="all">
+</head>
+<body ng-app="tweet-timeline">
+<div class="container" ng-controller="tweet-search">
+<h1 class="header">Tweet Search Timeline</h1>
+<hr>
+<div class="input-group input-group-lg search-bar" ng-submit="getTweets()">
+<input type="text" ng-model="search" class="form-control" placeholder="Search..">
+<div class="input-group-btn">
+<button type="submit" ng-click="getTweets()" class="btn btn-success search-button"><span style="padding-left:30px; padding-right:30px;" class="glyphicon glyphicon-search"></span></button>
+</div>
+</div>
+<h4 class="header" style="margin-top:20px;" ng-hide="loading"><center>Search results for "<span>{{searchTerm}}</span>"</center></h4>
```

Comment:
Line is longer than allowed. (132 > 129)

*LineLengthBear, severity NORMAL, section `default`.*